scss/_variables.scss

```diff
@@ -383,6 +383,7 @@ $enable-validation-icons:     true !default;
 $enable-negative-margins:     false !default;
 $enable-deprecation-messages: true !default;
 $enable-important-utilities:  true !default;
+$enable-debugger-classes:     true !default;
```

Comment:
default value looks `true`

Reply:
Yep, it's done on purpose at the moment to be able to test it on the deploy preview but I think it will go to `false` before getting merged.